packages/react-radio/src/stories/RadioGroupBestPractices.md

```diff
@@ -0,0 +1,23 @@
+## Best practices
+
+### Do
+
+- **Give people a choice.**
+  Make sure to give people the option to not make a choice. For example, include a "None" option if no answer is required.
+
+- **Choose a default.**
+  Select the safest, most secure, and private option as the default. If safety and security aren't factors, select the most likely or convenient option.
+
+- **Keep labels short and action-oriented.**
+  Use a phrase for the label, rather than a full sentence.
+
+- **Use sentence case.**
+  Use sentence style capitalization — only capitalize the first word.
```

Comment:
Nit: this description is a bit repetitive. What do you think about:
```suggestion
  Only capitalize the first word like you would in a sentence.
```

Reply:
Sure. These Best Practices were copied out of the visual spec, but I think your wording is clearer.